x-pack/plugins/security_solution/cypress/tasks/alerts_detection_rules.ts

```diff
@@ -217,9 +217,10 @@ export const waitForRulesTableToBeAutoRefreshed = () => {
 };
 
 export const waitForPrebuiltDetectionRulesToBeLoaded = () => {
-  cy.get(LOAD_PREBUILT_RULES_BTN).should('not.exist');
-  cy.get(RULES_TABLE).should('exist');
-  cy.get(RULES_TABLE_REFRESH_INDICATOR).should('not.exist');
+  // Wait up to 5 minutes for the rules to load as in CI containers this can be very slow
```

Comment:
i hope this doesn't end up taking this long 😬

Reply:
Same here, but I think it depends on the CI container. I noticed the container does additional CI calls for REST requests around o11y features so the container could be doing packet inspections or pushing more things or it could be throttling connections for all I know, so I gave it a liberal number.